price_matching_system/templates/user.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.html" %}
+
+{% block content %}
+
+<div class="row mt-5">
+    <div class="col-md-4 mx-auto p-5" style="min-width: 500px; border: solid black">
```

Comment:
can we use two columns to save vertical space ?

Reply:
UI can be decided again for sure.

Reply:
changed to two columns